Src/library/memory_analyzer/singletons/ma_singleton_factory.e

```diff
@@ -69,20 +69,6 @@ feature  -- Singletons
 			system_util_not_void: Result /= Void
 		end
 
-	main_window: MA_WINDOW
```

Comment:
Was there a reason to extract this code in MA_SHARED_MAIN_WINDOW? Couldn't just update and use this class instead?